lab_1_classify_by_unigrams/main.py

```diff
@@ -84,6 +127,34 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    if isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict):
```

Comment:
return None here

Reply:
I have "return None" at the end of the cycle. I tried to place it in a different way, but got mistakes with returns (there was something like  "you cannot return value and None").

Reply:
.

Reply:
done